uni-freiburg-geschichte.csl

```diff
@@ -0,0 +1,502 @@
+<?xml version="1.0" encoding="utf-8"?>
+<style xmlns="http://purl.org/net/xbiblio/csl" class="note" version="1.0" initialize-with="." name-as-sort-order="all" default-locale="de-DE">
+  <info>
+    <title>Uni Freiburg Geschichte</title>
+    <title-short>Uni Freiburg</title-short>
+    <id>http://www.zotero.org/styles/uni-freiburg-geschichte</id>
+    <link href="http://www.zotero.org/styles/uni-freiburg-geschichte" rel="self"/>
+     <link href="https://mittelalter2.geschichte.uni-freiburg.de/lehre/proseminarheft-mittelalter" rel="documentation"/>
+    <category citation-format="note"/>
+    <category field="humanities"/>
+    <updated>2020-02-12T14:08:32+00:00</updated>
+  </info>
```

Comment:
Add this before </info>:
    <rights license="http://creativecommons.org/licenses/by-sa/3.0/">This work is licensed under a Creative Commons Attribution-ShareAlike 3.0 License</rights>

Reply:
thanks for the help